Library/Homebrew/cmd/--env.rb

```diff
@@ -1,20 +1,128 @@
 #:  * `--env`:
 #:    Show a summary of the Homebrew build environment.
 
+
 require "extend/ENV"
 require "build_environment"
 
+
 module Homebrew
+  
+  MAX_ITERATIONS = 10
+  VALID_SHELLS = %w(
+    sh csh tcsh bash ksh mksh oksh
+    pdksh zsh fish
+  )
+  DEFAULT_SHELL = "sh"
+
+
```

Comment:
Only need one newline here (and elsewhere). Please follow the existing style used in the file.